src/main/java/org/mamute/controllers/AnswerController.java

```diff
@@ -70,6 +65,9 @@ public void edit(@Load Answer original, MarkedText description, String comment)
 		
 		UpdateStatus status = original.updateWith(information);
 		answers.save(original);
+		List<Attachment> attachmentsLoaded = attachments.load(attachmentsIds);
+		original.removeAttachments();
```

Comment:
it's easier to remove them all and then add the selected attachments than to find out the difference between the existent files and the new ones being created.